src/accelerate/launchers.py

```diff
@@ -118,21 +128,29 @@ def train(*args):
                     "inside your training function. Restart your notebook and make sure no cells initializes an "
                     "`Accelerator`."
                 )
-
-            if torch.cuda.is_initialized():
-                raise ValueError(
-                    "To launch a multi-GPU training from your notebook, you need to avoid running any instruction "
-                    "using `torch.cuda` in any cell. Restart your notebook and make sure no cells use any CUDA "
-                    "function."
-                )
-
             # torch.distributed will expect a few environment variable to be here. We set the ones common to each
             # process here (the other ones will be set be the launcher).
             with patch_environment(
                 world_size=num_processes, master_addr="127.0.01", master_port=use_port, mixed_precision=mixed_precision
             ):
+                # First dummy launch
+                if debug or os.environ.get("ACCELERATE_DEBUG_MODE", "false").lower() == "true":
```

Comment:
Yes, just that here. Let's not add a new arg, no?

Reply:
Done. I feel like this is common enough that a `debug` param directly would be better, but we can see how much use it gets (with the added docstring) during testing before fully commiting